src/main/java/lotto/domain/LottoList.java

```diff
@@ -12,22 +11,22 @@
     public LottoList(int buyCount) {
         lottoMachine = new LottoMachine();
         for (int i = 0; i < buyCount; i++) {
-            lottoList.add(lottoMachine.getAutoLotto());
+            lottoList.add(lottoMachine.generateAutoLotto());
         }
     }
 
     public WinningStatistics getWinning(WinningLottoNumbers winningNumbers) {
-        List<Rank> list = new ArrayList<>();
+        List<Rank> lottoRankList = new ArrayList<>();
         for (Lotto lotto : lottoList) {
             int numberOfWinnings = 0;
             numberOfWinnings = getNumberOfWinnings(lotto, numberOfWinnings, winningNumbers.getWinningLottoNumbers());
```

Comment:
```suggestion
            int numberOfWinnings = getNumberOfWinnings(lotto, winningNumbers.getWinningLottoNumbers());
```

```java
    private int getNumberOfWinnings(Lotto lotto, Lotto winningLotto) {
        return (int) lotto.getLottoNumbers().stream()
            .filter(lottoNumber -> winningLotto.getLottoNumbers().contains(lottoNumber))
            .count();
    }
```
매서드 파라미터를 3개이하로 줄려 로직을 간소화 해보는 것은 어떨가요?